Projects/Addons/Sources/Views/ChangeAddonSummaryScreen.swift

```diff
@@ -23,34 +23,33 @@ extension ChangeAddonViewModel {
     func asQuoteSummaryViewModel(changeAddonNavigationVm: ChangeAddonNavigationViewModel) -> QuoteSummaryViewModel {
 
         let newPremium =
-            changeAddonNavigationVm.changeAddonVm.selectedSubOption?.price
+            changeAddonNavigationVm.changeAddonVm.selectedQuote?.price
 
         let vm = QuoteSummaryViewModel(
             contract: [
                 .init(
-                    id: self.contractInformation?.contractId ?? "",
-                    displayName: changeAddonNavigationVm.changeAddonVm.contractInformation?.contractName ?? "",
-                    exposureName: changeAddonNavigationVm.changeAddonVm.contractInformation?.activationDate
-                        .localDateString ?? "",
+                    id: self.contractId ?? "",
+                    displayName: self.selectedQuote?.productVariant.displayName ?? "",
+                    exposureName: self.activationDate?.localDateString ?? "",
                     newPremium: newPremium,
                     currentPremium: nil,
-                    documents: self.contractInformation?.documents ?? [],
+                    documents: self.selectedQuote?.productVariant.documents ?? [],
                     onDocumentTap: { document in
 
                     },
-                    displayItems: self.contractInformation?.displayItems ?? [],
-                    insuranceLimits: self.contractInformation?.insurableLimits ?? [],
+                    displayItems: [], /* TODO: ADD */
```

Comment:
Not sure if these will be included in AddonVariant?

Reply:
We will not have that property at all I think